blockchain/blocks/src/gossip_block.rs

```diff
@@ -7,13 +7,27 @@ use forest_encoding::tuple::*;
 use fvm_ipld_encoding::Cbor;
 
 /// Block message used as serialized `gossipsub` messages for blocks topic.
-#[derive(Clone, Debug, Serialize_tuple, Deserialize_tuple)]
+#[derive(Clone, Debug, PartialEq, Serialize_tuple, Deserialize_tuple)]
 pub struct GossipBlock {
     pub header: BlockHeader,
     pub bls_messages: Vec<Cid>,
     pub secpk_messages: Vec<Cid>,
 }
 
+impl quickcheck::Arbitrary for GossipBlock {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let cid = Cid::new_v1(
+            u64::arbitrary(g),
+            cid::multihash::Multihash::wrap(u64::arbitrary(g), &[u8::arbitrary(g)]).unwrap(),
+        );
+        Self {
+            header: BlockHeader::arbitrary(g),
+            bls_messages: vec![cid],
+            secpk_messages: vec![cid],
```

Comment:
We want an arbitrary number of message cids.

Reply:
When the tipset PR is merged, I'll make this change

Reply:
Changed and fixed